modules/stic_Web_Forms/Catcher/EventInscription/EventInscriptionMailer.php

```diff
@@ -174,6 +190,37 @@ public function sendUserMail($templateId, $lang = null)
             $lang);
     }
 
+    /**
+     * Function to parse the email
+     *
+     * @param $templateId id of the template
+     * @param $account data of the account
+     * @param $payment data of the payment
+     * @param $replacementObjects array with the object
+     * @param $lang
+     * @return void
+     */
+    public function parsingEmail($templateId, $payment, $replacementObjects, $account, $lang){
+        // Function to get the object if there is a payment
+        if (!empty($payment)) {
+            $replacementObjects[] = $payment;
+            if ($payment->load_relationship('stic_payments_stic_payment_commitments')) {
+                $relatedBeans = $payment->stic_payments_stic_payment_commitments->getBeans();
+                foreach ($relatedBeans as $fpBean) {
+                    $replacementObjects[] = $fpBean;
+                }
+            }
+        }
+
+        // Parse the template
+        $GLOBALS['log']->debug('Line ' . __LINE__ . ': ' . __METHOD__ . ":  Parsing template [{$templateId}]...");
+
+        if (false === parent::parseEmailTemplateById($templateId, $replacementObjects, $account, $lang)) {
```

Comment:
Esta invocación no tiene los mismos parámetros que la definición de la función. Además para invocar la función, si no es que estamos sobre-escribiendo la función en la clase y queremos llamar a la de la clase padre, no se usa el parent::, si no $this->